awswrangler/redshift.py

```diff
@@ -1,4 +1,5 @@
 """Amazon Redshift Module."""
+# pylint: disable=too-many-lines
```

Comment:
Any reason why this is applied at the file level instead of the offending method?

Reply:
Yes, this is actually regarding too many lines **in module**:
`awswrangler/redshift.py:1:0: C0302: Too many lines in module (1507/1500) (too-many-lines)`